app/assets/stylesheets/layout_components/accordions.scss

```diff
@@ -52,27 +52,46 @@
 }
 
 .accordion {
+  border-top: 1px solid $smoke;
+  border-bottom: 1px solid $smoke;
+  background: none;
+
   &__header {
-    background: $brand-blue-light;
     overflow: hidden;
     display: block;
+    padding: $base-padding/1.5;
+    padding-left: $base-padding;
+    font-weight: 600;
```

Comment:
Is this weight doing anything? I can disable it in the DOM and things seem to stay the same.